python/cudf/cudf/core/column/string.py

```diff
@@ -3624,6 +3624,37 @@ def findall(self, pat: str, flags: int = 0) -> SeriesOrIndex:
         data = libstrings.findall(self._column, pat, flags)
         return self._return_or_inplace(data)
 
+    def find_re(self, pat: str, flags: int = 0) -> SeriesOrIndex:
+        """
+        Find first occurrence of pattern or regular expression in the
+        Series/Index.
+
+        For details, see :cpp:func::`find_re`.
```

Comment:
My apologies, I suggested this line in my previous review, but it shouldn't be here. We should reference the libcudf function in pylibcudf, not cudf classic.
```suggestion
```